### ISSUE TITLE
Is there deploy.prototxt available? 

### ISSUE THREAD

**Issue by bryanyzhu** [liuzhuang13/DenseNetCaffe#8]
@liuzhuang13 @gaohuang Thanks for the great work. I want to apply the pre-trained model to my own dataset, but it achieved worse results than other architectures. I think I may generate the deploy.prototxt file wrong. Specifically, I don't know how to write batch_norm, scale, dropout layer in a deploy.prototxt to make sure it functions properly during test phase. Could you share your deploy.prototxt as a reference? I appreciate any help.

**Comment by liuzhuang13** [comment]
@bryanyzhu    Sorry for the late response. Our pretrained DenseNet-121 in Caffe has just been released here https://github.com/liuzhuang13/DenseNet#imagenet-and-pretrained-models

Thanks for your interest

**Comment by bryanyzhu** [comment]
Thanks a lot. I will close the issue now.
